build_tools/bazel/build_core.sh

```diff
@@ -13,7 +13,9 @@
 # Looks at environment variables and uses CI-friendly defaults if they are not
 # set.
 # IREE_LLVMAOT_DISABLE: Do not run tests that require LLVM-AOT. Default: 0
-# IREE_VULKAN_DISABLE: Do not run tests that require Vulkan. Default: 1
+# IREE_VULKAN_DISABLE: Do not run lit tests that require Vulkan. Default: 0
+# IREE_VULKAN_DISABLE_RUNTIME: Do not run runtime tests that require Vulkan.
```

Comment:
This was always the intent of `IREE_VULKAN_DISABLE`, I think. The Vulkan *target* shouldn't really need to be disabled anywhere